packages/react-examples/src/react-divider/Divider/Divider.stories.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import { Divider, DividerProps } from '@fluentui/react-divider';
+import * as classes from '../react-divider.stories.scss';
```

Comment:
let's try to remove scss from anything converged

Reply:
This is only for the Storybook examples, and not the actual component itself. I am using it for example layout etc. Can we have a quick discussion to see if it can still be used for this purpose?

Reply:
You could use css in js to achieve a similar thing so I think you could remove scss entirely that way.

Reply:
Updated